packages/admithub-lightbox/package.js

```diff
@@ -8,12 +8,17 @@ Package.onUse(function(api) {
   api.use([
     'accounts-base',
     'stylus',
-    'telescope:core@0.24.0',
+    'telescope:core',
```

Comment:
let's make sure that we don't accidentally upgrade to the next telescope version and break something down the line.  Though this change isn't a blocker.